diff --git a/third_party/python/importlib_metadata/importlib_metadata/__init__.py b/third_party/python/importlib_metadata/importlib_metadata/__init__.py
index 089fca97de..37302cd549 100644
--- a/third_party/python/importlib_metadata/importlib_metadata/__init__.py
+++ b/third_party/python/importlib_metadata/importlib_metadata/__init__.py
@@ -588,4 +588,7 @@ def requires(distribution_name):
     return distribution(distribution_name).requires
 
 
-__version__ = version(__name__)
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    __version__ = '1.5.0' # hack, enter correct version here
diff --git a/third_party/python/jsonschema/jsonschema/__init__.py b/third_party/python/jsonschema/jsonschema/__init__.py
index 6b630cdfbb..d22bac1fe9 100644
--- a/third_party/python/jsonschema/jsonschema/__init__.py
+++ b/third_party/python/jsonschema/jsonschema/__init__.py
@@ -31,4 +31,6 @@ try:
     from importlib import metadata
 except ImportError: # for Python<3.8
     import importlib_metadata as metadata
-__version__ = metadata.version("jsonschema")
+    
+# __version__ = metadata.version("jsonschema")
+__version__ = '3.2.0' # hack, enter correct jsonschema version here.
